core/state/statedb.go

```diff
@@ -74,6 +74,15 @@ func (m *mutation) isDelete() bool {
 	return m.typ == deletion
 }
 
+type arbFiltered int
+
+const (
+	txFiltered arbFiltered = 1 + iota
```

Comment:
I think there should be another value in the enum for the zero value.
Maybe `unfiltered` or `notFiltered`

Reply:
added it